msamp/megatron/layers.py

```diff
@@ -13,6 +13,14 @@
 from msamp.common.tensor import ScalingTensor
 from msamp.operators.gemm import Gemm
 
+import os
+
+USE_W_SIMU_FP4 = bool(int(os.getenv('USE_W_SIMU_FP4', 0)))
```

Comment:
Shall we make the naming clearer? Like using `weight` and `activation` instead of `W` and `A`